internal/resources/policy/type/security/scope_schema.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package security
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	policyclustermodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/policy/cluster"
+	policyclustergroupmodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/policy/clustergroup"
+	policyorganizationmodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/policy/organization"
+	scoperesource "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/resources/policy/scope"
+)
+
+var scopeSchema = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "Scope for the policy",
+	Required:    true,
+	ForceNew:    true,
+	MaxItems:    1,
+	MinItems:    1,
```

Comment:
can some validations be added here ?

Reply:
Since it is just a block (implemented as a list) we can add only limits to the number of items which is already in place. 

As for the validations, those are here: https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/84f222049084b0a6c679eaaadca5cd8fc2a90a9e/internal/resources/policy/type/security/scope_schema.go#L113

Reply:
And we have specified the same in the customized diff handler https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/84f222049084b0a6c679eaaadca5cd8fc2a90a9e/internal/resources/policy/type/security/resource_security_policy.go#L35